src/MessagePack.UnityClient/Assets/Scripts/MessagePack/Resolvers/DynamicObjectResolver.cs

```diff
@@ -1264,6 +1264,9 @@ void OnNotFound()
                     il.Emit(OpCodes.Br, readNext);
                 }
 
+#if NET_STANDARD_2_0
```

Comment:
What has this to do with the rest of the change?

Reply:
I used NET_STANDARD_2_0 derective to compile MessagePack for my project.
Method AutomataDictionary.AutomataDictionary(...) is not defined for this derective.
See:
https://github.com/neuecc/MessagePack-CSharp/blob/8fe5fc94669f9cc1d3feb2bdbc5d5523e54d0e9a/src/MessagePack.UnityClient/Assets/Scripts/MessagePack/Internal/AutomataDictionary.cs#L134

So I added thid condition here to be able to compile project with NET_STANDARD_2_0 derective.